Resources/Prototypes/_Nuclear14/Recipes/Lathes/misc.yml

```diff
@@ -83,4 +83,25 @@
   result: Matchbox
   completetime: 1
   materials:
-    Wood: 100
\ No newline at end of file
+    Wood: 100
+
+- type: latheRecipe
```

Comment:
These recipes probably need added to the entity you make them on. E.g forge or workbench